frontend/src/components/common/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Switch from './Switch';
+
+/**
+ * 공용 Switch 컴포넌트
+ */
+const meta: Meta<typeof Switch> = {
+  title: 'Common/Switch',
```

Comment:
공용 컴포넌트를 다루고 있는 디렉토리명은 소문자 `common`이고, 지금까지 공용 컴포넌트를 만들 때에도 이 부분은 폴더 대소문자 그대로 `common`으로 적었으니, 이 부분도 소문자 `common`으로 적어줄 수 있을까?

```suggestion
  title: 'common/Switch',
```